packages/web-app-admin-settings/src/views/Users.vue

```diff
@@ -2,7 +2,6 @@
   <div>
     <app-template
       ref="template"
-      :loading="loadResourcesTask.isRunning || !loadResourcesTask.last"
```

Comment:
why was this removed?

Reply:
It just moved from there to the current view, because the filter triggers the user loading. It should only re-load the users-list, not the entire page.

Reply:
can we do the same with the other views and purge it from the app template?

Reply:
Yes we should do that, but I'd rather do it in a separate PR to keep this PR here about the filter only.